crypto/property/properties.ebnf

```diff
@@ -0,0 +1,17 @@
+(* https://bottlecaps.de/rr/ui *)
+
+Definition
+      ::= PropertyName ( '=' Value )? ( ',' PropertyName ( '=' Value )? )*
+Query ::= ( '-'? PropertyName | PropertyName ( '=' | '!=' ) Value )
+          ( ',' ( '-'? PropertyName | PropertyName ( '=' | '!=' ) Value ) )*
+Value    ::= NumberLiteral
+        | StringLiteral
+StringLiteral ::= QuotedString | UnquotedString
```

Comment:
I think unquotedstring is a very big mistake.  It makes it really hard to add keywords to the query language in the future.  Look at perl's "barewords" evolution, for example.